src/tools/pub/package-layout.md

```diff
@@ -58,14 +58,11 @@ enchilada/
     style.css
 {% endprettify %}
 
-\* The `.dart_tool/` directory exists after you've run `pub get`. 
```

Comment:
I think it would be nice to keep this quick reference here as readers may not go further down the page.

Could add something like to reference that later section as well:

```markdown
To learn more, 
see [Project specific caching for tools](#project-specific-caching-for-tools).
```

Reply:
Ahh, shoot, it's not letting me commit this suggestion.